pkg/addons/translate.go

```diff
@@ -324,6 +324,28 @@ var Translations = map[string][]struct {
 			Template: "{{ .Config.Images.WebConsole }}",
 		},
 	},
+	"Group.user.openshift.io/customer-admins": {
+		{
+			Path: jsonpath.MustCompile("$.users"),
+			F: func(cs *api.OpenShiftManagedCluster) (interface{}, error) {
+				if cs.Config.CustomerAdminPasswd != "" {
```

Comment:
@kargakis, checking whether the password is empty instead of `derived.runningundertest`.

Reply:
This seemed to do the trick:
```
KUBECONFIG=kc oc get groups
NAME               USERS
customer-admins    customer-cluster-admin
customer-readers   customer-cluster-reader
```

Reply:
Woot!
```
Ran 9 of 15 Specs in 10.899 seconds
SUCCESS! -- 9 Passed | 0 Failed | 0 Pending | 6 Skipped
--- PASS: TestExtended (10.90s)
PASS
ok  	github.com/openshift/openshift-azure/test/e2e	10.919s
2018/10/24 18:11:50 Container test in pod e2e-azure completed successfully
```